lib/common.dart

```diff
@@ -1,6 +1,22 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:jiffy/jiffy.dart';
 
-const DATE_FORMAT = 'yyyy-MM-dd';
+// The date at time formats to use for Jiffy DateTime formatting
+const DISPLAY_DATE_FORMAT = 'yyyy-MM-dd';
+const DISPLAY_TIME_FORMAT = 'HH:mm:ss'; // 24-hour time format
+
+String displayNeatTimestamp(DateTime? dateTime) {
```

Comment:
Why making it null and account for null in it in such a way when we can make it just DateTime?